llmfoundry/utils/builders.py

```diff
@@ -168,12 +167,12 @@ def build_scheduler(name: str,
 def build_tokenizer(
         tokenizer_name: str,
         tokenizer_kwargs: Dict[str, Any]) -> PreTrainedTokenizerBase:
-    if tokenizer_name == 'openai':
-        return OpenAITokenizerWrapper(**tokenizer_kwargs)
-    else:
-        os.environ['TRANSFORMERS_NO_ADVISORY_WARNINGS'] = '1'
-        os.environ['TOKENIZERS_PARALLELISM'] = 'false'
+    os.environ['TRANSFORMERS_NO_ADVISORY_WARNINGS'] = '1'
+    os.environ['TOKENIZERS_PARALLELISM'] = 'false'
```

Comment:
hm, a little weird to be setting env vars in this function...out of scope for this PR, but we should probably be setting env vars another place

Reply:
I think this is actually the right place. You don't need to set them if you are not using huggingface tokenizers, so we set them right before we instantiate one of those.